bin/kano-updater

```diff
@@ -115,8 +115,9 @@ def schedule_install(gui=False, confirm=True, splash_pid=None):
         confirm = False
 
         title = "Updater"
-        description = "Urgent updates have been downloaded! We'll need to install these as soon" \
-                      " as possible to ensure the best experience."
+        description = "Urgent updates have been downloaded! We'll need to" \
```

Comment:
do we want to `i18n` this string?

Reply:
We should probably do another pass of the i18n state of the Updater. Will do that in another PR